junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/DefaultArgumentConverter.java

```diff
@@ -184,4 +182,26 @@ public Object convert(String source, Class<?> targetType) throws Exception {
 		}
 	}
 
+	static class StringToJavaMiscConverter implements StringToObjectConverter {
```

Comment:
Duplicate code with StringToPrimitiveConverter. Maybe we can merge with StringToPrimitiveConverter class by changing its name?

Reply:
I didn't quite follow you.

Can you please expound?

Reply:
@sbrannen  If you look at `StringToPrimitiveConverter ` and this class. They are very similar. Maybe, we can use a single class instead of two. I am not sure what to do.

Reply:
@hisener I think the same argument could be made for `StringToJavaTimeConverter`. I personally think it's fine to have `StringToJavaMiscConverter` be separate from `StringTo(Primitive|JavaTime)Converter`. They could all share an abstract class if it would save lines of code. WDYT @sbrannen?

Reply:
I agree: let's keep them separate.

I don't think the minor code duplication is an issue here (yet).